application/helpers/filebin_helper.php

```diff
@@ -1,5 +1,30 @@
 <?php
 
+function expiration_duration($duration)
+{
+	$total = $duration * 60 * 60 * 24;
+	$days = floor($total / 86400);
+	$total -= $days * 86400;
+	$hours = floor($total / 3600);
+	$total -= $hours * 3600;
+	$minutes = floor($total / 60);
+	$times = array($days, $hours, $minutes);
+	$suffixes = array(' day', ' hour', ' minute');
+	$expiration = "";
+
+	for ($i = 0; $i < 3; $i++) {
```

Comment:
This should use ``count($times)`` or ``count($suffixes)`` instead of a hardcoded 3.